### PR TITLE
Update code-optimizations-vs-extension.md

### PR DESCRIPTION
**Pull request by johnbilliris**
With the Application Insights Code optimisation extension for Visual Studio, you are able to use the application insight's Application ID to connect to it from within Visual Studio.

Also, Visual Studio version 17.12 is not yet supported.

### CODE REVIEW

**Comment by Court72** [comment]
@hhunter-ms

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by hhunter-ms — APPROVED** [review]

File: articles/azure-monitor/insights/code-optimizations-vs-extension.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ This article guides you through using the extension with GitHub Copilot in Visua
 
 ## Prerequisites
 
-- [Install versions 17.10 or 17.11 of Visual Studio](https://visualstudio.microsoft.com/downloads/)
+- [Install versions 17.10 or 17.11 of Visual Studio](https://visualstudio.microsoft.com/downloads/). (**Note:** Version 17.12 of Visual Studio is not yet supported.)
```
Comment (Court72) [inline articles/azure-monitor/insights/code-optimizations-vs-extension.md:21]:
```suggestion
- [Install versions 17.10 or 17.11 of Visual Studio](https://visualstudio.microsoft.com/downloads/). 
   > [!NOTE]
   > Version 17.12 of Visual Studio is not yet supported.
```

File: articles/azure-monitor/insights/code-optimizations-vs-extension.md [hunk 2]
```diff
@@ -48,10 +49,10 @@ Once installed, the Code Optimizations extension (preview) introduces an agent c
 
 1. In Visual Studio, open the repo holding your .NET application with Code Optimizations enabled. 
 1. Open the GitHub Copilot chat. 
-1. Invoke the extension by executing the following command, replacing the placeholder with your own Application Insights resource name.
+1. Invoke the extension by executing the following command, replacing the placeholder with your own Application Insights resource name or with the Application Insights Application ID
```
Comment (Court72) [inline articles/azure-monitor/insights/code-optimizations-vs-extension.md:52]:
```suggestion
1. Invoke the extension by executing the following command, replacing the placeholder with your own Application Insights resource name or with the Application Insights Application ID.
```

**Comment by Court72** [comment]
@hhunter-ms,

I'll merge this PR based on your approval. If this article requires further changes, please open a separate pull request.

**Pull request merged**